components/dialog/integration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Integration
+page_title: Dialog Integration
+description: Discover the Blazor Dialog Integration and explore the examples.
+slug: dialog-integration
+tags: telerik,blazor,dialog,integration
+published: True
+position: 9
+---
+
+# Blazor Dialog Integration
```

Comment:
If I look only at the title, I would think that the article's topic is how the Dialog integrates _into_ other components. Try being more specific.
```suggestion
# Integrating Components into the Blazor Dialog
```